src/store/flow/block.ts

```diff
@@ -81,14 +81,21 @@ export const mutations: MutationTree<IFlowsState> = {
     Vue.delete(findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config!, key)
   },
   block_updateConfigByPath(state, {blockId, path, value}: {blockId: string, path: string, value: object | string}) {
-    // todo: this might still break reactivity
-    // Make nested assignment reactive
```

Comment:
Thanks for address this TODO, could we add a JsDoc mentioning we'd like to `make nested assignment reactive` ?
(that will explain the chunk usage)